dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.json

```diff
@@ -0,0 +1,476 @@
+    },
+    {
+     "type": "comment",
+     "name": "activities_you_liked",
+     "title": "What were the one or two things you liked most about the activities you did in this workshop and why?",
+     "cols": 6,
+     "rows": 6
+    },
+    {
+     "type": "comment",
+     "name": "activities_youd_change",
+     "title": "What are the one or two things you would change about the activities you did in this workshop?\t",
+     "cols": 60,
+     "rows": 6
+    },
+    {
+     "type": "paneldynamic",
+     "name": "facilitator_feedback_panel",
+     "title": "Facilitator Feedback",
+     "description": "Please leave feedback about your facilitator(s).",
+     "templateElements": [
+      {
+       "type": "matrix",
+       "name": "facilitator_effectiveness",
+       "title": "During my workshop, {panel.facilitatorName} did the following:",
+       "columns": [
+        {
+         "value": "1",
+         "text": "Strongly Disagree"
+        },
+        {
+         "value": "2",
+         "text": "Disagree"
+        },
+        {
+         "value": "3",
+         "text": "Slightly Disagree"
+        },
+        {
+         "value": "4",
+         "text": "Neutral"
+        },
+        {
+         "value": "5",
+         "text": "Slightly Agree"
+        },
+        {
+         "value": "6",
+         "text": "Agree"
+        },
+        {
+         "value": "7",
+         "text": "Strongly Agree"
+        }
+       ],
+       "rows": [
+        {
+         "value": "demonstrated_knowledge",
+         "text": "Demonstrated knowledge of the curriculum."
+        },
+        {
+         "value": "built_equitable",
+         "text": "Built and sustained an equitable learning environment in our workshop."
+        },
+        {
+         "value": "on_track",
+         "text": "Kept the workshop and participants on track."
+        },
+        {
+         "value": "productive_discussions",
+         "text": "Supported productive workshop discussions."
+        },
+        {
+         "value": "ways_equitable",
+         "text": "Helped me see ways to create and support an equitable learning environment for my students."
+        },
+        {
+         "value": "healthy_relationship",
+         "text": "Demonstrated a healthy working relationship with their co-facilitator (if applicable)."
+        }
+       ]
+      },
+      {
+       "type": "comment",
+       "name": "facilitator_did_well",
+       "title": "What were two things {panel.facilitatorName} did well?"
+      },
+      {
+       "type": "comment",
+       "name": "facilitator_could_improve",
+       "title": "What were two things {panel.facilitatorName} could do better?"
+      }
+     ],
+     "templateTitle": "Facilitator {panel.facilitatorPosition} of {panel.numFacilitators} - {panel.facilitatorName}",
```

Comment:
we'll need add the facilitatorPosition and numFacilitators variables if you need them, but we can add those!

Reply:
I think they're nice to have as an aid to the reader.

Reply:
I assuming I should convert to _ rather than camelCase for these as well?

Reply:
for these we currently have them as camelCase so you can leave them.